src/shaders/standard.js

```diff
@@ -69,11 +69,11 @@ module.exports.Shader = registerShader('standard', {
   update: function (data) {
     this.updateMaterial(data);
     utils.material.updateMap(this, data);
-    if (data.normalMap) { utils.material.updateDistortionMap('normal', this, data); }
```

Comment:
This guards no longer needed?

Reply:
The guards prevented these textures from being removed. Similar to how the main (diffuse) map is handled, the implementation checks if the texture is added, changed or removed, or if only the texture properties are updated.